Xamarin.Forms.Platform.UAP/WindowsBasePlatformServices.cs

```diff
@@ -28,10 +29,12 @@ internal abstract class WindowsBasePlatformServices : IPlatformServices
 	{
 		const string WrongThreadError = "RPC_E_WRONG_THREAD";
 		readonly CoreDispatcher _dispatcher;
+		readonly UISettings _uiSettings = new UISettings();
 
 		protected WindowsBasePlatformServices(CoreDispatcher dispatcher)
 		{
 			_dispatcher = dispatcher ?? throw new ArgumentNullException(nameof(dispatcher));
+			_uiSettings.ColorValuesChanged += UISettingsColorValuesChanged;
```

Comment:
Could we unsubscribe this event?

Reply:
Nothing in this class is disposed or unsubscribed. I think this lives for the entire lifecycle of the application, right? So it only goes away when the app stops and then is cleaned up with everything else?